packages/app/src/components/feature/widget/ConfigurableMapView/index.tsx

```
@@ -0,0 +1,130 @@
+import { Coordinate } from "@model/map";
+import { memo, useState } from "react";
+
+import CurrentPosition from "./currentPosition";
+import DomainMarkers from "./domainMarkers";
+import HeadPolygon from "./headPolygon";
+import type { NaverMapMarkerProps } from "./marker";
+// import NaverMapMarker from "./marker";
+
+const DEFAULT_ZOOM = 14;
+
+export interface PolylineOptions extends NaverMapPathOverlayProps {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)